postman/collection-dir/paypal/event.prerequest.js

```diff
@@ -0,0 +1,33 @@
+const path = pm.request.url.toString();
+const isPostRequest = pm.request.method.toString() === "POST";
+const isPaymentCreation = path.match(/\/payments$/) && isPostRequest;
+
+if (isPaymentCreation) {
+  try {
+    const request = JSON.parse(pm.request.body.toJSON().raw);
+
+    // Attach profile_id
+    const profile_id = isPaymentCreation
+      ? pm.collectionVariables.get("payment_profile_id")
+      : pm.collectionVariables.get("payout_profile_id");
+    request["profile_id"] = profile_id;
```

Comment:
It is better to remove these lines since we already receive `profile_id` in payment response.

Reply:
Done